### PR TITLE
Add Alt keyboard modifier to VUI to disable snapping to grids

### PR DESCRIPTION
**Pull request by traprog**
Implementation is the same as that of the timeline. That is to say, Alt is hard coded.

### CODE REVIEW

**Comment by bmatherly** [comment]
LGTM. Personally, I prefer this over the checkbox in the grid submenu because:
* It is consistent with the timeline
* The user can engage/disengage it easily without stopping a drag in progress

File: src/qml/filters/corners/vui.qml [hunk 1]
```diff
@@ -251,8 +251,10 @@ Shotcut.VuiBase {
                         cursorShape: Qt.SizeFDiagCursor
                         drag.target: parent
                         onPositionChanged: {
-                            corner1Handle.x = snapX(corner1Handle.x + handleOffset) - handleOffset;
-                            corner1Handle.y = snapY(corner1Handle.y + handleOffset) - handleOffset;
+                            if(!(mouse.modifiers & Qt.AltModifier)) {
```
Comment (ddennedy) [inline src/qml/filters/corners/vui.qml:254]: please respect our code formatting: space between `if` and `(`

**Review by ddennedy — APPROVED** [review]
I do want to include this since it can be said that the shortcut to suspend snapping is not working here and can be considered a bug. Please do fix the minor code format, however.

**Comment by ddennedy** [comment]
Thank you

**Pull request merged**